packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -65,19 +69,33 @@ export const PaymentConfirmation = ({
         data-testid="payment-confirmation"
       >
         <header>
-          <img
-            className="circled-check"
-            src={circledCheckbox}
-            alt="circled checkbox"
-          />
-          <Localized id="payment-confirmation-thanks-heading">
-            <h2>Thank you!</h2>
+          {accountExists ? (
+            <>
+              <img
+                className="circled-check"
+                src={circledCheckbox}
+                alt="circled checkbox"
+              />
+            </>
+          ) : (
+            <>
+              <img src={checkmarkIcon} alt="checkmark icon" />
+              <img
+                className="email-sent-icon"
+                src={emailSentIcon}
+                alt="email sent"
+              />
+            </>
+          )}
+
+          <Localized id="payment-confirmation-thanks-heading-2">
+            <h2>Thanks, now check your email!</h2>
           </Localized>
           <Localized
-            id="payment-confirmation-thanks-subheading"
-            vars={{ email, product_name }}
+            id="payment-confirmation-thanks-subheading-2"
+            vars={{ email }}
           >
-            <p>{`A confirmation email has been sent to ${email} with details on how to get started with ${product_name}.`}</p>
+            <p>{`You'll receive an email at ${email} with instructions for setting up your account as well as  your payment details.`}</p>
           </Localized>
```

Comment:
We probably need to differentially show these messages based on `accountExists` too. Right now I see the passwordless strings when I sign up with an existing FxA account locally.